src/babel/index.ts

```diff
@@ -131,6 +136,11 @@ export default function ({
             const importSpecifierStringLiteral = t.stringLiteral(
               importSpecifier
             );
+            const processPath = process.cwd();
+            const modulePath = require.resolve(importSpecifier);
```

Comment:
Running this within `react-loosely-lazy-app` fails for me, when enabling the babel plugin within the repository. I think you need to take more care into making sure this works and doesn't break functionality. Though correct me if I'm wrong, because I expected based on your README that your component repository should not work, but app should since it is meant to mimic a regular consumer.

Reply:
Can I ask how you added the babel plugin and what the failure was? If I add the plugin into the webpack config like so

```
  module: {
    rules: [
      {
        test: /\.js$/,
        exclude: /node_modules/,
        use: {
          loader: 'babel-loader',
          options: {
            babelrc: false,
            presets: [
              ['@babel/preset-env', { modules: false }],
              '@babel/preset-react'
            ],
            plugins: [
              ['@babel/plugin-proposal-class-properties'],
              '@babel/plugin-proposal-export-namespace-from',
              '@babel/plugin-syntax-dynamic-import',
              '@babel/plugin-syntax-import-meta',
              [
                'react-loosely-lazy/babel-plugin',
                {
                  client: true
                }
              ]
            ]
          }
        }
      }
    ]
  },
```

I am able to build and run the demo without error.

Reply:
This should be fixed now. So what you need to do is

- checkout `react-loosely-lazy-app` and `npm install`
- build `react-loosely-lazy` 
- copy the built webpack and babel plugins from `react-loosely-lazy` to `react-loosely-lazy-app/node_modules/react-loosely-lazy`
- Run the app in demo mode and see what's going on
- Build the app and inspect the chunks/manifest